routemaster/server/tests/test_endpoints.py

```diff
@@ -39,16 +39,17 @@ def test_enumerate_state_machines(client, app_config):
     ]}
 
 
-def test_create_label(client, app_config):
+def test_create_label(client, app_config, mock_test_feed):
     label_name = 'foo'
-    state_machine = list(app_config.config.state_machines.values())[0]
+    (state_machine,) = app_config.config.state_machines.values()
```

Comment:
I'm fairly sure the parentheses around `(state_machine,)` aren't needed.

Reply:
dc4a46056d40a3b0705745e00e15b06615f84de6